core/src/env/call/create_builder.rs

```diff
@@ -0,0 +1,338 @@
+// Copyright 2019-2020 Parity Technologies (UK) Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use crate::{
+    env,
+    env::{
+        call::{
+            utils::{
+                EmptyArgumentList,
+                ReturnType,
+                Set,
+                Unset,
+                Unwrap,
+            },
+            ExecutionInput,
+        },
+        EnvError,
+        EnvTypes,
+    },
+};
+use core::marker::PhantomData;
+
+/// Contracts that can be contructed from an `AccountId`.
+///
+/// # Note
+///
+/// This is needed because of conflicting implementations of `From<T> for T`
+/// in the generated code of `ink_lang`.
+pub trait FromAccountId<T>
+where
+    T: EnvTypes,
+{
+    /// Creates the contract instance from the account ID of the already instantiated contract.
+    fn from_account_id(account_id: <T as EnvTypes>::AccountId) -> Self;
+}
+
+/// Builds up contract instantiations.
+#[derive(Debug)]
+pub struct CreateParams<E, Args, R>
```

Comment:
Should it not still be `Instantiate` rather than `Create` so it matches the name from the contracts pallet? Also all comments say `instantiate`

Reply:
The main reason I chose `Create` over `Instantiate` was that it is shorter and simpler to type. Blame me for my inability to use a keyboard. :D